src/pages/management/configuration-api/index.mdx

```diff
@@ -2304,9 +2304,9 @@ For properties without an explicit value assignment, the method will return the
 |                 |                                                                               |
 | --------------- | ----------------------------------------------------------------------------- |
 | Method URL      | `https://api.livechatinc.com/v3.4/configuration/action/list_group_properties` |
-| Required scopes | `properties.group.value--all:ro`*                                             |
+| Required scopes | `properties.group.value--all:rw`*                                             |
```

Comment:
`ro` scope is not available now, so the change was made for the `rw' scope as it can be used instead.